Gems/Profiler/Code/Source/ImGuiCpuProfiler.cpp

```diff
@@ -126,6 +111,19 @@ namespace Profiler
         m_mainThreadId = AZStd::hash<AZStd::thread_id>{}(AZStd::this_thread::get_id());
     }
 
+    float ImGuiCpuProfiler::TicksToMs(double ticks)
+    {
+        const AZStd::sys_time_t ticksPerSecond = m_ticksPerSecondFromFile > 0 ? m_ticksPerSecondFromFile : AZStd ::GetTimeTicksPerSecond();
+        // Note: converting to microseconds integer before converting to milliseconds float
+        AZ_Assert(ticksPerSecond >= 1000, "Error in converting ticks to ms, expected ticksPerSecond >= 1000");
+        return static_cast<float>((ticks * 1000) / (ticksPerSecond / 1000)) / 1000.0f;
+    }
+
+    float ImGuiCpuProfiler::TicksToMs(AZStd::sys_time_t ticks)
+    {
+        return TicksToMs(static_cast<double>(ticks));
+    }
```

Comment:
Ms is normally meant to mean "milliseconds", while Us is meant to mean "microseconds"

But instead of needing a ticks to microsecond, you could just have a function that performs a returns `AZStd::chrono::duration<float, AZStd::chrono::micro>` 
```suggestion
    float ImGuiCpuProfiler::TicksToMs(double ticks)
    {
        const AZStd::sys_time_t ticksPerSecond = m_ticksPerSecondFromFile > 0 ? m_ticksPerSecondFromFile : AZStd ::GetTimeTicksPerSecond();
        // Note: converting to microseconds integer before converting to milliseconds float
        AZ_Assert(ticksPerSecond >= 1000, "Error in converting ticks to ms, expected ticksPerSecond >= 1000");
        return static_cast<float>((ticks * 1000) / (ticksPerSecond / 1000)) / 1000.0f;
    }

    float ImGuiCpuProfiler::ToFloatMicroseconds(AZStd::sys_time_t ticks)
    {
        return AZStd::chrono::duration<float, AZStd::chrono::micro>(m_timeMicroseconds).count();
    }
```

Reply:
This is meant to convert from integer microseconds to float milliseconds